examples/ecs_with_ec2/extra.tf

```diff
@@ -0,0 +1,2466 @@
+/*
```

Comment:
Please clean-up all comments and test code before creating a PR. If it contains good example codes for different scenarios, create separate examples in separate directories.